stablehlo/transforms/Passes.td

```diff
@@ -96,3 +96,13 @@ def VhloToVersionPass : Pass<"vhlo-to-version"> {
            "The target version. Must be a version of the form #.#.# or 'current'.">,
   ];
 }
+
+def LegalizeChlo :
+    InterfacePass<"stablehlo-legalize-chlo", "mlir::FunctionOpInterface"> {
+  let summary = "Legalizes from CHLO ops flow to StableHLO and Shape ops";
+}
+
+def StableHLOCanonicalize :
```

Comment:
1. Can we split this pass into a separate PR for review? Will make it easier to see what's a CHLO specific infra and what isn't. I.e. I'm not sure if there is a test to check that this pass is working properly yet? Would be easier to review / give feedback as a standalone PR.
2. Per feedback on: [openxla-discuss](https://groups.google.com/a/openxla.org/g/openxla-discuss/c/httnobiya7U) Let's rename this pass `StablehloAggressiveSimplificationPass` and `stablehlo-aggressive-simplification`, and we can gradually move some to native canonicalizers.

Reply:
Sorry, I don't have access to Google Groups.I can't see what's in the discuss.

Reply:
Ah! If you share an email address I can CC you into the thread. Feel free to send me a message from your email to gleasonk@google[dot]com, if interested :)

To sum up the feedback, there is strong support for these passes in StableHLO, but disagreement on which of these passes is a canonicalization vs an opinionated transform, so it was suggested we name it "StablehloAggressiveSimplificationPass" and gradually migrate agreed upon patterns to canonicalizations.

Reply:
Thanks!